draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -60,14 +59,20 @@ the receiving side, the QUIC stack is free to surface the stream reset to the
 application immediately, without providing any stream data it has received for
 that stream.
 
-Some applications running on top of QUIC send an identifier at the beginning of
-the stream in order to associate that stream with a specific subcomponent of the
-application. For example, WebTransport ({{!WEBTRANSPORT}}) uses a
+Some applications running on top of QUIC use bytes at the beginning of the
+stream to communicate critical information related to that stream. For example,
+to associate the stream with a specific subcomponent of the application.
+Concretely, WebTransport ({{?WEBTRANSPORT=I-D.ietf-webtrans-http3}}) uses a
```

Comment:
```suggestion
stream to communicate critical information related to that stream.
For example, WebTransport ({{?WEBTRANSPORT=I-D.ietf-webtrans-http3}}) uses a
```

Reply:
I was trying to avoid chaining "For example," in consecutive sentences. How about

```suggestion
stream to communicate critical information related to that stream. For example,
to associate the stream with a specific subcomponent of the application.
For instance, WebTransport ({{?WEBTRANSPORT=I-D.ietf-webtrans-http3}}) uses a
```